Emergence/Contexts/Presenting Locations/LocationsHost.swift

```diff
@@ -0,0 +1,52 @@
+import JavaScriptCore
+import UIKit
+
+struct LocationsHost {
+
+    let featured = ["new-york", "london", "los-angeles", "paris", "berlin", "miami", "san-francisco", "hong-kong", "milan", "sao-paolo", "tokyo"]
+    let cities: [Location]
+
+    // Gnarly function to grab locations from local cache
+    init? () {
+        let bundle = NSBundle.mainBundle()
+        guard let citiesPath = bundle.pathForResource("cities", ofType: "js")  else {
+            return nil
+        }
+
+        do {
+            var citiesString = try String(contentsOfFile: citiesPath, encoding: NSUTF8StringEncoding)
+            citiesString = citiesString.stringByReplacingOccurrencesOfString("module.exports", withString: "var cities")
```

Comment:
we're importing a nom module, so make it global scope then we can easily access the data.